entity-service-change-event-api/src/main/java/org/hypertrace/entity/service/change/event/api/EntityChangeEventGenerator.java

```diff
@@ -0,0 +1,17 @@
+package org.hypertrace.entity.service.change.event.api;
+
+import java.util.Map;
+import org.hypertrace.entity.data.service.v1.Entity;
+
+/** The interface Entity change event generator. */
+public interface EntityChangeEventGenerator {
```

Comment:
I'd probably just put this in the same package as the impl. This isn't something we need published, but the protos we do. We can make the interface public and the impl package protected to keep the same separation we have here (just need a public builder or static build method somewhere).

Reply:
Nah, that's fine.